### ISSUE TITLE
Configuring with any autoconf version other then 2.69 fails

### ISSUE THREAD

**Issue by madebr** [ianlancetaylor/libbacktrace#74]
Hello,

Running `autoreconf`, with autoconf 2.71, fails with the following error:
```
configure.ac:33: error: Please use exactly Autoconf 2.69 instead of 2.71.
```

The cause are the following lines:
https://github.com/ianlancetaylor/libbacktrace/blob/4f57c999716847e45505b3df170150876b545088/config/override.m4#L30-L48

What is the reason only 2.69 is allowed? Does the configure script use undocumented features?
Would it be ok to remove these lines?

**Comment by Nekto89** [comment]
There was similar issue https://github.com/ianlancetaylor/libbacktrace/issues/71#issuecomment-813538749

**Comment by madebr** [comment]
Thanks for the link. I'm closing this one.

**Comment by ianlancetaylor** [comment]
I've reconsidered this.  While the version of libbacktrace in the GCC sources should require autoconf 2.69, there's no special reason to require 2.69 in the standalone libbacktrace.  I've commented out the code (copied from GCC) that requires exactly 2.69.

That said using autoconf 2.71 reports some warnings about obsolete autoconf commands.  I don't plan to fix those unless and until GCC starts avoiding them.
